apolloschurchapp/android/app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,7 @@
 
           <!-- Handle urls starting with "http://www.example.com/products" -->
           <data android:scheme="https"
-                android:host="@string/APP_DATA_URL"
+                android:host="rivervalley-herokuapp-com.global.ssl.fastly.net"
```

Comment:
these should be heroku links right?

Reply:
I actually tested this fastly one several times (which was already in place) and didn't notice any issues with it. Can definitely change all these references to the heroku instance.

Reply:
I went ahead and changed it!  Tested as well and all looks good. I'm gonna go ahead and push this to master. I feel good about these changes.